### DISCUSSION TITLE
2.8.0 更新日志

### DISCUSSION THREAD

**Discussion by LTCatt** [Meloong-Git/PCL#4170] [讨论]
· NeoForge 兼容与自动安装（#2244，PR @Pigeon0v0，感谢 @3gf8jv4dv）
· 支持编译、运行 PCL 开源代码
　　修改了 PCL 存储库指引文件，以允许 Fork、修改代码和有条件地分发衍生版本
　　修改活跃橙隐藏主题描述，允许向主存储库提交 Pull Request 以解锁活跃橙
· 将内存分配上限限制为 32G
· 添加追随者披风的中文翻译（#3885、#4138，PR @wuyuncheng-26、@Deep-Dark-Forest）
· 自定义事件 [启动游戏] 允许启动当前的 MC（#3965，@WSXYT，PR @youzi-2333）
· 添加部分 Dragonwell Java 的识别（#4006，@acaciachan）
· Mod/整合包 下载增加 1.21 筛选（#4031，@WForst-Breeze）
· [启动游戏前进行内存优化] 支持版本独立设置（#4058，PR @youzi-2333）
· 优化 Java 搜索代码（#4093，PR @SALTWOOD）
· 为 1.14~15 自动安装重新启用 OptiFabric Origins 安装提示（#4153，@zec896219）
· 只有一首音乐时允许右键从头开始重新播放（#4155，PR @youzi-2333）
· 修复了以下 14 个 Bug
　　特定情况下删除/重命名版本后设置信息异常（#3120，@xiaoniaoawa）
　　资源下载的翻页键可能显示为未激活样式（#3319，@WForst-Breeze）
　　第三方登录注册按钮跳转有误（#3819、#4151，@allMagicNB，PR @Mxmilu666）
　　无法使用 URL 打开已安装的基岩版（#3985，@MCSteve123，PR @JingHai-Lingyun）
　　版本独立设置中的“自动选择合适的 Java”无效（#3898，@FlameFire26）
　　使用老版本导致个性化设置丢失（#3941，@JingHai-Lingyun）
　　正式版中点击“输入解锁码”导致崩溃（#3990，@Light-Beacon）
　　CurseForge 整合包的光影被错误地放到 mods 文件夹（#3995，@ciaotanjiajun）
　　使用特定名称的自定义皮肤无法进入游戏（#4013，@skilahello，PR @Silverteal）
　　极度罕见情况下启动下载管理线程出错（#4021，@chengzi-sc）
　　可能错误地将 Mod 更新为另一个加载器的版本（#4064，@spf2333）
　　可能自动分配超过 64G 的内存，导致游戏崩溃
　　播放音乐时错误地显示“左键播放”，应为“左键暂停”
　　安装 Forge 时可能会漏下载部分支持库文件

<hr /><em>This discussion was created from the release <a href='https://github.com/Hex-Dragon/PCL2/releases/tag/2.8.0'>2.8.0</a>.</em>

**Comment by Deep-Dark-Forest** [comment]
@LTCatt 这里少了个#4018（
![Screenshot_20240706_180032.jpg](https://github.com/Hex-Dragon/PCL2/assets/131975043/1106e3ae-c2a6-46ff-8218-ec6b9ee92c6e)

**Reply by LTCatt** [reply]
如果 issue 和一个 PR 关联，现在只给出 issue 的编号。issue 右边栏的 development 以及时间线上都可以查到对应的 PR……

**Reply by Deep-Dark-Forest** [reply]
报意思，才发现（
